### PR TITLE
Fix typos in docs

### PR DESCRIPTION
**Pull request by kianmeng**
Found via this command:

    codespell -L afterall

### CODE REVIEW

File: docs/src/api/class-page.md [hunk 1]
```diff
@@ -2435,7 +2435,7 @@ User can inspect selectors or perform manual steps while paused. Resume will con
 the place it was paused.
 
 :::note
-This method requires Playwright to be started in a headed mode, with a falsy [`option: headless`] value in
+This method requires Playwright to be started in a headed mode, with a falsely [`option: headless`] value in
```
Comment (jfgreffier) [inline docs/src/api/class-page.md:2438]: `falsy` is correct https://developer.mozilla.org/en-US/docs/Glossary/Falsy

**Review by debs-obrien — APPROVED** [review]
thanks for your contribution :)

**Review by mxschmitt — APPROVED** [review]

**Comment by mxschmitt** [comment]
For the future you need to run "npm ci && npm run build" to update the generated files. I did it for you.

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**